indico/web/client/styles/partials/_main.scss

```diff
@@ -222,36 +225,117 @@ nav.main-breadcrumb {
 
 /* Styles for the session bar at the top right corner */
 
-div.session-bar {
+#session-bar {
   @include font-family-title();
 
-  margin: 0.5em 0.5em 0 0;
+  flex: none;
+  display: flex;
+  align-items: center;
+  gap: 0.1em;
 
-  .toolbar {
```

Comment:
When you activate the language dropdown, there is this white triangle that partially covers the `M` in "My language":

<img width="412" height="743" alt="Image" src="https://github.com/user-attachments/assets/4d43bf60-6235-40f3-a64c-6c8429cce420" />